gatsby-node.js

```diff
@@ -220,6 +220,16 @@ exports.createPages = async function ({ actions, graphql }) {
     toPath: "/#products",
     isPermanent: true,
   });
+  // Create redirects for all pages that used to be in the old `/about` directory:
```

Comment:
Is this for backwards compatibility? i.e. will we want to remove this after most folks' caches have expired?